packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.tsx

```diff
@@ -37,6 +37,7 @@ import { mirrorHorizontalPlacement } from '@instructure/ui-position'
 import generateStyle from './styles'
 import generateComponentTheme from './theme'
 import { DrawerLayoutContext } from '../index'
+import { BidirectionalProps } from '@instructure/ui-i18n/src/bidirectional'
```

Comment:
I think we should export the `BidirectionalProps` from the index of the `ui-i18n` package

Reply:
I agree, we shouldn't import anything from /src

Reply:
yep, fixed